staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -36,14 +54,15 @@ type leaseManager struct {
 	prevLeaseExpirationTime time.Time
 	// The period of time in seconds and percent of TTL that each lease is
 	// reused. The minimum of them is used to avoid unreasonably large
-	// numbers. We use var instead of const for testing purposes.
+	// numbers.
 	leaseReuseDurationSeconds int64
 	leaseReuseDurationPercent float64
+	count                     int64
```

Comment:
Can we have a more meaningful name?

leaseAttachedObjectCount or sth like that?